app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
cool the test looks great!

sorry I forgot to mention before that I agree with your suggestion to 

>  What do you think about putting the content_for back into the view, but also moving the partial out of shared, since it isn't actually shared at this point?

want to do that now?